### ISSUE TITLE
When lowering minDiff slider in generator the query of the tricks doesn't change

### ISSUE THREAD

**Issue by bastislack** [bastislack/highline-freestyle#169]
When I put both sliders to 11 and then reduced the minDiff still the tricks were only from level 11

**Comment by bastislack** [comment]
Same goes for Maximum Difficulty when increased

**Comment by bastislack** [comment]
Seems like the setState is not taking effect as also the log statement in useEffect is only executed in two out of four cases (decreasing maxDiff and increasing minDiff)

**Comment by rorystephenson** [comment]
When I create a combo and then move the slider the following error occurs:

```javascript
react_devtools_backend.js:4061 Warning: findDOMNode is deprecated in StrictMode. findDOMNode was passed an instance of Handle which is inside StrictMode. Instead, add a ref directly to the element you want to reference. Learn more about using refs safely here: https://reactjs.org/link/strict-mode-find-node
    at div
    at Handle (webpack://highline-freestyle/./node_modules/rc-slider/es/Handle.js?:37:90)
    at div
    at ComponentEnhancer (webpack://highline-freestyle/./node_modules/rc-slider/es/common/createSlider.js?:60:92)
    at div
    at form
    at div
    at ComboGenerator (webpack://highline-freestyle/./src/components/generator/ComboGenerator.js?:32:5)
    at Routes (webpack://highline-freestyle/./node_modules/react-router/index.js?:276:5)
    at div
    at div
    at div
    at eval (webpack://highline-freestyle/./node_modules/react-div-100vh/dist/esm/index.js?:35:18)
    at div
    at div
    at div
    at Router (webpack://highline-freestyle/./node_modules/react-router/index.js?:209:15)
    at BrowserRouter (webpack://highline-freestyle/./node_modules/react-router-dom/index.js?:115:5)
    at App (webpack://highline-freestyle/./src/App.js?:47:81)
    at ReactPWAInstallProvider (webpack://highline-freestyle/./node_modules/react-pwa-install/es/index.js?:39:23)
```

**Comment by rorystephenson** [comment]
I think this is fixed by #176
